packages/@react-aria/color/src/useColorSlider.ts

```diff
@@ -61,6 +62,32 @@ export function useColorSlider(props: ColorSliderAriaOptions, state: ColorSlider
     inputRef
   }, state);
 
+  let {step, pageSize} = state.value.getChannelRange(channel);
+  let {keyboardProps} = useKeyboard({
+    onKeyDown(e) {
```

Comment:
Actually do we need this stuff now? `useSliderThumb` seems to handle firing onChange and onChangeEnd just fine in the ColorSlider stories when I remove this locally

Reply:
Don't we still need it for `pageUp`/`pageDown` events?

Reply:
I think you're right. PageStep seems to be handled by `useSliderThumb`.

Reply:
right, perhaps this addition of useKeyboard was originally included to override useSliderThumb's behavior so it wouldn't fire onChangeEnd if the value didn't change but now we are ok with that behavior for parity for now